packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -100,16 +100,38 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
                   entry: [sendUpdate(), 'clearError'],
                   invoke: {
                     src: 'signUp',
+                    onDone: [
+                      {
+                        cond: 'shouldSkipConfirm',
+                        target: 'skipCheck',
+                        actions: ['setUser'],
+                      },
+                      {
+                        target: 'resolved',
+                        actions: ['setUser', 'setCredentials'],
+                      },
+                    ],
+                    onError: {
+                      target: 'idle',
+                      actions: 'setRemoteError',
+                    },
+                  },
+                },
+                skipCheck: {
+                  tags: ['checking'],
+                  invoke: {
```

Comment:
I wasn't really sure if I needed to add this

`              entry: [sendUpdate(), 'clearError'],`

When I did it would cause an issue where the `Create Account` button would light up for a few seconds before it moved to the sign in page. So I removed this.

Reply:
Your approach looks good, `sendUpdate()` isn't really needed here because the parent machine doesn't need to know about it.